modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -293,52 +294,79 @@
 import static org.apache.ignite.lifecycle.LifecycleEventType.BEFORE_NODE_START;
 
 /**
- * Ignite kernal.
- * <p/>
- * See <a href="http://en.wikipedia.org/wiki/Kernal">http://en.wikipedia.org/wiki/Kernal</a> for information on the
- * misspelling.
+ * This class represents an implementation of the extended main Ignite API (called {@link IgniteEx}). It also controls
```

Comment:
Should we mention printing metrics, detected long operations and a starvation?

Reply:
Added.